dev/tidb-in-kubernetes/maintain/backup-and-restore.md

```diff
@@ -139,31 +139,29 @@ kubectl get pvc -n <namespace> -l app.kubernetes.io/component=backup,pingcap.com
 
 ### Pump 缩容
 
-缩容 Pump 需要挨个先将 Pump 节点从集群中下线之后操作 `helm upgrade` 进行缩容。
+缩容 Pump 需要**挨个**将 Pump 节点从集群中下线，然后操作 `helm upgrade` 将对应的 Pump Pod 删除。
```

Comment:
```suggestion
缩容 Pump 需要先将 Pump 节点从集群中**逐一**下线，然后操作 `helm upgrade` 将对应的 Pump Pod 删除。
```

Reply:
To scale in Pump, for each Pump node, make the node offline and delete the corresponding Pump Pod instead of making all the unwanted nodes offline and deleting the corresponding Pump Pods. 

To make it clearer, how about "缩容 Pump 需要先将单个 Pump 节点从集群中下线，然后操作 `helm upgrade` 将对应的 Pump Pod 删除，并对每个节点重复上述步骤。"?

Reply:
@juliezhang1112 Agree.